code/modules/organs/external/head.dm

```diff
@@ -12,7 +12,7 @@
 	joint = "jaw"
 	amputation_point = "neck"
 	encased = "skull"
-	artery_name = "cartoid artery"
+	artery_name = "carotid artery"
```

Comment:
wtf I'm having a Berenstain Bears moment. I would have sworn on any god you liked that it was cartoid.